core-kotlin-modules/core-kotlin-collections-4/src/test/kotlin/com/baeldung/maxvalue/MaxValueArrayTest.kt

```diff
@@ -0,0 +1,95 @@
+package com.baeldung.maxvalue
+
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatExceptionOfType
+import org.junit.jupiter.api.Test
+
+class MaxValueArrayTest {
+
+    @Test
+    fun givenNonEmptyArray_whenGetMaxOrNull_thenMaxValueIsReturned() {
+
```

Comment:
In this and the other tests, remove the empty line at the beginning. One of the guidelines prescribes not to have empty lines after an opening curly brace or before a closing one.